resources/hiveclients/python/hiveserver1-client.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# imports
+import sys
+import hive_utils
+from hive_service.ttypes import HiveServerException
+
+# get the input parameters
+hiveHost = sys.argv[1]
```

Comment:
Maybe you could check that the argv array has the right number of argument (no more, no less) and raise and error (and print the usage hint) in that case.